# On branch master
# Changes to be committed:
#   (use "git reset HEAD <file>..." to unstage)
#
#	modified:   GL/glx/glxcmds.c
#
# Untracked files:
#   (use "git add <file>..." to include in what will be committed)
#
#	export-DoCreateGLXPixmap.patch
#	extramodes
#	log
diff --git a/GL/glx/glxcmds.c b/GL/glx/glxcmds.c
index 91c80b2..e201311 100644
--- a/GL/glx/glxcmds.c
+++ b/GL/glx/glxcmds.c
@@ -1176,7 +1176,7 @@ DoCreateGLXDrawable(ClientPtr client, int screenNum, XID fbconfigId,
     return Success;
 }
 
-static int
+int
 DoCreateGLXPixmap(ClientPtr client, int screenNum, XID fbconfigId,
 		  XID drawableId, XID glxDrawableId)
 {
